lib/grinder_tools.dart

```diff
@@ -139,10 +139,10 @@ Future runProcessAsync(GrinderContext context, String executable,
  * A default implementation of an `init` task. This task verifies that the grind
  * script is executed from the project root.
  */
-void defaultInit(GrinderContext context) {
+void defaultInit([GrinderContext context]) {
```

Comment:
Since I think most (all?) of the changes so far for 0.7.0 are non-breaking, I vote we keep the required params for now (and ignore their values), and since it's impossible to deprecate the required params, we just doc-comment that they will be removed soon (and that they are ignored), and then in the next breaking version after 0.7.0 we remove the all the context params (even the optional ones).